src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -206,6 +207,11 @@ private void initializeVcfWriter() {
         final Set<VCFHeaderLine> hInfo = new LinkedHashSet<>();
         hInfo.addAll(getHeaderForVariants().getMetaDataInInputOrder());
 
+        // need AC, AN and AF since output if set filtered genotypes to no-call
```

Comment:
`If setFilteredGenotypesToNoCall is true, then allele counts will be recomputed and need to be included in the header`